packages/xstate-react/src/createXStateContext.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { createContext, useContext } from 'react';
+import {
+  InterpreterFrom,
+  MachineOptions,
+  StateFrom,
+  StateMachine
+} from 'xstate';
+import { useActor } from './useActor';
+import { useInterpret } from './useInterpret';
+import { useSelector } from './useSelector';
+
+export function createXStateContext<
+  TMachine extends StateMachine<any, any, any, any>
+>(machine: TMachine) {
+  const context = createContext({} as InterpreterFrom<TMachine>);
```

Comment:
it would be nice if we'd accept a `displayName` parameter and we'd set it on this created context - it will display better in the React devtools etc

Reply:
Yes, agree - do you think it should be required or optional?

Reply:
> Yes, agree - do you think it should be required or optional?

Perhaps optional, and use the machine ID for a default

Reply:
👍